app/controllers/items_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ItemsController < ApplicationController
+
+  def index
+    if params[:active] == "true"
```

Comment:
This if statement is a great way to customize your index page! You could also roll the "items" action from the Merchants controller into it as well. That would be the most "conventional" way to do it.